vassal-app/src/main/java/VASSAL/build/module/map/Scroller.java

```diff
@@ -38,10 +38,10 @@
    * The attribute name for whether to use arrow keys
    * instead of number keypad.  Should be one of ALWAYS, NEVER, or PROMPT
    */
-  public static final String USE_ARROWS = "useArrows";
-  public static final String ALWAYS = "always";
-  public static final String NEVER = "never";
-  public static final String PROMPT = "prompt";
+  public static final String USE_ARROWS = "useArrows"; //NON-NLS
+  public static final String ALWAYS = "always"; //NON-NLS
+  public static final String NEVER = "never"; //NON-NLS
+  public static final String PROMPT = "prompt"; //NON-NLS
 
   protected String usingArrows = PROMPT;
```

Comment:
I don't think this is your problem Brian, but I just can't work out how this class is supposed to work. usingArrows is maintained as a String attribute with the ALWAYS/NEVER/PROMPT, but there is no way that usingArrows can ever be changed as Scroller is not Configurable. The USE_ARROWS preference is actually Boolean and is checked explicitly in the keyPressed() method.

I think the whole ALWAYS/NEVER/PROMPT stuff can just be thrown away and just set the Boolean configurer. Thoughts?

Reply:
Oh right... WEIRD. I clearly did NOT stare deeply into that class when I came through...

Reply:
For now, I'm happy if we just do the i18n and leave any other changes for another day...

Reply:
Well see what you think of this. Also there is that weird "NoEcho" thing which looks like it might have done something once but doesn't seem to do anything now? It's like long ago someone came and cut a chunk out of this class.

I can always just revert this to the "mostly inert" form it was already in.

Reply:
Nah, getting rid of clutter is good. Waiting for the build...